### ISSUE TITLE
Exception when saving synced field to form

### ISSUE THREAD

**Issue by jmauzyk** [verbb/formie#867]
**Description**
Getting the following exception when trying to save a synced field on a form:
`PDOException: SQLSTATE[22P02]: Invalid text representation: 7 ERROR:  invalid input syntax for type integer: "sync:682"`

**Steps to reproduce**
1. Click *Add existing fields*, select an existing field, then click *Add 1 as syned field*.
2. Click *Save*.
3. Error notification appears: *Unable to save form.* Exception above is printed in `web.log`.

### Form settings
- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.5.16
- Craft version: Pro 3.7.40.1
- Multi-site: No

**Additional context**
- DB: PostgreSQL 14.2

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.16",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.17](https://github.com/verbb/formie/releases/tag/1.5.17)
